core/src/main/scala/org/apache/spark/util/collection/ChainedBuffer.scala

```diff
@@ -32,7 +32,7 @@ private[spark] class ChainedBuffer(chunkSize: Int) {
   assert(math.pow(2, chunkSizeLog2).toInt == chunkSize,
     s"ChainedBuffer chunk size $chunkSize must be a power of two")
   private val chunks: ArrayBuffer[Array[Byte]] = new ArrayBuffer[Array[Byte]]()
-  private var _size: Int = _
+  private var _size: Long = _
```

Comment:
I think that we should initialize this with 0 instead of `_`.  They're actually equivalent, but the former is clearer.